cico_setup.sh

```diff
@@ -41,14 +61,13 @@ tag_push() {
 
 push_image() {
     local image_name
-    local image_repository
     local short_commit
     local push_registry
+    local full_image_name
     image_name=$(make get-image-name)
-    image_repository=$(make get-image-repository)
     short_commit=$(git rev-parse --short=7 HEAD)
-    push_registry="push.registry.devshift.net"
-    
+    push_registry=$(make get-registry)
```

Comment:
@anmolbabu Now the push registry is "registry.centos.org" . And we know you can't push to that registry. Also, we are logging into the Devshift registry afterwards (At line 72). Isn't this conflicting?

Reply:
@dev-gaur I had a chat with Aditya and his point was that, we need to have our images on registry.devshift 
registry.centos is also ok
but registry.devshift is kind of we must..

Reply:
Anyway our centos-pipeline integration already builds images for us..

Reply:
Hey, I'm aware that we need to maintain images on the devshift registry inorder for OSIO to recognise them.

What I'm pointing out is that since 
''' push_registry = ${make get-registry} '''
Would effectively convert to 
''' push_registry = "registry.centos.org" ''' ,

On this line,

https://github.com/anmolbabu/supervisor/blob/7251da5d45e98eedae9a9d846b3e947c6c6126c7/cico_setup.sh#L71

The login would be attempted at the centos registry and not the devshift registry.

Correct me if I'm wrong.